src/app/components/auth/login/login.component.ts

```diff
@@ -1,34 +1,60 @@
-import { Component } from '@angular/core';
-import { FormBuilder, FormGroup, Validators } from '@angular/forms';
-import { Router } from '@angular/router';
-import { RouterModule } from '@angular/router';
-import { ReactiveFormsModule } from '@angular/forms';
-import { InputTextModule } from 'primeng/inputtext';
-import { CardModule } from 'primeng/card';
-import { ButtonModule } from 'primeng/button';
+import { Component } from "@angular/core";
+import { FormBuilder, FormGroup, Validators } from "@angular/forms";
+import { Router } from "@angular/router";
+import { RouterModule } from "@angular/router";
+import { ReactiveFormsModule } from "@angular/forms";
+import { InputTextModule } from "primeng/inputtext";
+import { CardModule } from "primeng/card";
+import { ButtonModule } from "primeng/button";
+import { HttpClient } from "@angular/common/http";
+import { AuthService } from "src/app/services/auth.service";
 
 @Component({
-  selector: 'app-login',
-  standalone: true,
-  imports: [RouterModule, ReactiveFormsModule, CardModule, ButtonModule, InputTextModule],
-  templateUrl: './login.component.html',
-  styleUrl: './login.component.scss',
+	selector: "app-login",
+	standalone: true,
+	imports: [RouterModule, ReactiveFormsModule, CardModule, ButtonModule, InputTextModule],
+	templateUrl: "./login.component.html",
+	styleUrl: "./login.component.scss",
 })
 export class LoginComponent {
-  loginForm: FormGroup;
+	constructor(
+		private fb: FormBuilder,
+		private router: Router,
+		private authService: AuthService,
+		private http: HttpClient,
+	) {
+		this.loginForm = this.fb.group({
+			email: ["", [Validators.required, Validators.email]],
+			password: ["", [Validators.required, Validators.minLength(6)]],
+		});
+	}
+	loginForm: FormGroup;
+	errorMessage: string | null = null;
 
-  constructor(private fb: FormBuilder, private router: Router) {
-    this.loginForm = this.fb.group({
-      email: ['', [Validators.required, Validators.email]],
-      password: ['', [Validators.required, Validators.minLength(6)]]
-    });
-  }
+	login() {
```

Comment:
let's start to add access modifiers and return type for all the methods in the app and 
```suggestion
	private login(): void {
```